Signal/src/ViewControllers/ConversationView/ConversationViewController.m

```diff
@@ -1388,11 +1401,13 @@ - (void)toggleDefaultKeyboard
         return;
     }
 
+    // FIXME inputAccessoryView - if using numeric keyboard, switch back to alpha after
+    // sending.
```

Comment:
?

Reply:
Forgot to re-enable this behavior. Done!